src/chrome/content/rules/Fefe.xml

```diff
@@ -31,17 +30,23 @@
 	^fefe.de doesn't exist.
 
 -->
+<ruleset name="Fefe.de (partial)" platform="Lets Encrypt">
+  <target host="blog.fefe.de" />
+        <target host="ptrace.fefe.de" />
+        <target host="www.fefe.de" />
+
+        <rule from="^http:"
+                to="https:" />
+</ruleset>
+
 <ruleset name="Fefe.de (partial)" platform="cacert" default_off="missing certificate chain">
 
 	<!--	Direct rewrites:
 				-->
-  <target host="blog.fefe.de" />
-	<target host="cvs.fefe.de" />
+  <target host="cvs.fefe.de" />
 	<target host="dl.fefe.de" />
 	<target host="a.mx.fefe.de" />
```

Comment:
`cvs` and `a.mx` are still problematic due to missing certificate chain, suggest to remove line 46 & 48.